sc8280xp.c

```diff
@@ -99,334 +111,334 @@ static struct debug_mux mc_cc = {
-	{ "disp1_cc_mdss_byte0_clk",				&gcc, 0x82, &disp1_cc, 0x19 },
-	{ "disp1_cc_mdss_byte0_intf_clk",			&gcc, 0x82, &disp1_cc, 0x1a },
-	{ "disp1_cc_mdss_byte1_clk",				&gcc, 0x82, &disp1_cc, 0x1b },
-	{ "disp1_cc_mdss_byte1_intf_clk",			&gcc, 0x82, &disp1_cc, 0x1c },
-	{ "disp1_cc_mdss_dptx0_aux_clk",			&gcc, 0x82, &disp1_cc, 0x23 },
-	{ "disp1_cc_mdss_dptx0_link_clk",			&gcc, 0x82, &disp1_cc, 0x1f },
-	{ "disp1_cc_mdss_dptx0_link_intf_clk",			&gcc, 0x82, &disp1_cc, 0x20 },
-	{ "disp1_cc_mdss_dptx0_pixel0_clk",			&gcc, 0x82, &disp1_cc, 0x24 },
-	{ "disp1_cc_mdss_dptx0_pixel1_clk",			&gcc, 0x82, &disp1_cc, 0x25 },
-	{ "disp1_cc_mdss_dptx0_usb_router_link_intf_clk",	&gcc, 0x82, &disp1_cc, 0x21 },
-	{ "disp1_cc_mdss_dptx1_aux_clk",			&gcc, 0x82, &disp1_cc, 0x31 },
-	{ "disp1_cc_mdss_dptx1_link_clk",			&gcc, 0x82, &disp1_cc, 0x2a },
-	{ "disp1_cc_mdss_dptx1_link_intf_clk",			&gcc, 0x82, &disp1_cc, 0x2b },
-	{ "disp1_cc_mdss_dptx1_pixel0_clk",			&gcc, 0x82, &disp1_cc, 0x26 },
-	{ "disp1_cc_mdss_dptx1_pixel1_clk",			&gcc, 0x82, &disp1_cc, 0x27 },
-	{ "disp1_cc_mdss_dptx1_usb_router_link_intf_clk",	&gcc, 0x82, &disp1_cc, 0x2c },
-	{ "disp1_cc_mdss_dptx2_aux_clk",			&gcc, 0x82, &disp1_cc, 0x32 },
-	{ "disp1_cc_mdss_dptx2_link_clk",			&gcc, 0x82, &disp1_cc, 0x2d },
-	{ "disp1_cc_mdss_dptx2_link_intf_clk",			&gcc, 0x82, &disp1_cc, 0x2e },
-	{ "disp1_cc_mdss_dptx2_pixel0_clk",			&gcc, 0x82, &disp1_cc, 0x28 },
-	{ "disp1_cc_mdss_dptx2_pixel1_clk",			&gcc, 0x82, &disp1_cc, 0x29 },
-	{ "disp1_cc_mdss_dptx3_aux_clk",			&gcc, 0x82, &disp1_cc, 0x37 },
-	{ "disp1_cc_mdss_dptx3_link_clk",			&gcc, 0x82, &disp1_cc, 0x34 },
-	{ "disp1_cc_mdss_dptx3_link_intf_clk",			&gcc, 0x82, &disp1_cc, 0x35 },
-	{ "disp1_cc_mdss_dptx3_pixel0_clk",			&gcc, 0x82, &disp1_cc, 0x33 },
-	{ "disp1_cc_mdss_esc0_clk",				&gcc, 0x82, &disp1_cc, 0x1d },
-	{ "disp1_cc_mdss_esc1_clk",				&gcc, 0x82, &disp1_cc, 0x1e },
-	{ "disp1_cc_mdss_mdp1_clk",				&gcc, 0x82, &disp1_cc, 0x12 },
-	{ "disp1_cc_mdss_mdp_clk",				&gcc, 0x82, &disp1_cc, 0x11 },
-	{ "disp1_cc_mdss_mdp_lut1_clk",				&gcc, 0x82, &disp1_cc, 0x16 },
-	{ "disp1_cc_mdss_mdp_lut_clk",				&gcc, 0x82, &disp1_cc, 0x15 },
-	{ "disp1_cc_mdss_non_gdsc_ahb_clk",			&gcc, 0x82, &disp1_cc, 0x3a },
-	{ "disp1_cc_mdss_pclk0_clk",				&gcc, 0x82, &disp1_cc, 0xf },
-	{ "disp1_cc_mdss_pclk1_clk",				&gcc, 0x82, &disp1_cc, 0x10 },
-	{ "disp1_cc_mdss_rot1_clk",				&gcc, 0x82, &disp1_cc, 0x14 },
-	{ "disp1_cc_mdss_rot_clk",				&gcc, 0x82, &disp1_cc, 0x13 },
-	{ "disp1_cc_mdss_rscc_ahb_clk",				&gcc, 0x82, &disp1_cc, 0x3c },
-	{ "disp1_cc_mdss_rscc_vsync_clk",			&gcc, 0x82, &disp1_cc, 0x3b },
-	{ "disp1_cc_mdss_vsync1_clk",				&gcc, 0x82, &disp1_cc, 0x18 },
-	{ "disp1_cc_mdss_vsync_clk",				&gcc, 0x82, &disp1_cc, 0x17 },
-	{ "disp1_cc_sleep_clk",					&gcc, 0x82, &disp1_cc, 0x46 },
-	{ "disp1_cc_xo_clk",					&gcc, 0x82, &disp1_cc, 0x45 },
-	{ "measure_only_mccc_clk", 				&gcc, 0xfeedbeef, &mc_cc, 0x50 },
```

Comment:
Neat! Now `mc_cc` no longer has a fake/useless gcc parent and mux value!

Reply:
Yes, that was one of the main reasons!